command/agent/job_endpoint.go

```diff
@@ -2060,3 +2060,48 @@ func validateEvalPriorityOpt(priority int) HTTPCodedError {
 	}
 	return nil
 }
+
+func (s *HTTPServer) JobsStatusesRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	args := structs.JobsStatusesRequest{}
+	if s.parse(resp, req, &args.Region, &args.QueryOptions) {
+		return nil, nil // seems whack
+	}
+	if smartOnly, err := parseBool(req, "smart_only"); err != nil {
+		return nil, err
+	} else if smartOnly != nil {
+		args.SmartOnly = *smartOnly
+	}
+	switch req.Method {
+	case http.MethodGet:
+		// GET requests will be treated as "get all jobs" but also with filtering and pagination and such
```

Comment:
Although it's against the spec, I'm _pretty_ sure Go's server will accept a body on GET anyways. So we're silently dropping the body here, and I'm not sure that's better or worse than returning an error that says "hey, use POST".